src/deion/common/constants.js

```diff
@@ -46,6 +46,13 @@ const PHASE_TEXT = {
     "8": "free agency",
 };
 
+const PLAYOFF_STATUS = {
```

Comment:
The PLAYOFF_STATUS constant translation really only make sense if it's persisted to the database, which it's currently not. So I'd get rid of that.

Reply:
Yeah makes sense. I'll do the conversion to string before returning the Standings results to the visual layer. I do like it initially being an INT so I can do MAX() functions, with some values superseding others